configure.ac

```diff
@@ -680,7 +680,7 @@ int main()
 ])],
 [ AC_DEFINE(RETURN_MACROS_WORK, 1) AC_MSG_RESULT([yes]) ],
 [ AC_DEFINE(RETURN_MACROS_WORK, 0) AC_MSG_RESULT([no]) ],
-[ AC_DEFINE(RETURN_MACROS_WORK, 0) AC_MSG_RESULT([no]) ]
```

Comment:
Can you revert this?  I still want the default to be off.

Reply:
`AC_RUN_IFELSE` can't be run when cross-compiling, so we have to update `action-if-cross-compiling` which is only used when cross-compiling: https://www.gnu.org/software/autoconf/manual/autoconf-2.64/html_node/Runtime.html
Without this change, `RETURN_MACROS_WORK` will always be set to 0 when cross-compiling.

Reply:
But what happens if someone is cross compiling onto an architecture that causes the return address code to segfault.  By forcing the yes it will be forcing a broken result.   You should be able to override the `RETURN_MACROS_WORK` default for your particular build.

Reply:
I think the appropriate way to do that is to alter the `settings.h` file and set `USE_RETURN_MACROS` to be 1.